js/main.js

```diff
@@ -23,8 +28,10 @@ function getPokemonEntry() {
       pokemon.name = allPokemon[i].pokemon_species.name;
       $row.appendChild(renderPokemonEntries(pokemon));
     }
+    xhr.addEventListener('error', function () {
+      alert('Sorry, there was an error connecting to the network! Please check your internet connection and try again.');
+    });
```

Comment:
If you define a function named `handleNetworkError` you can cut down on duplicating this alert.